auth0.ios.ts

```diff
@@ -20,14 +20,23 @@ export class Auth0Lock extends common.Auth0Lock{
             var page = frameModule.topmost().ios.controller;
 
             let lock: Lock = Lock.classic();
-
+            
             //Add scope
             if(this.options.scope){
                 var scopeItems = this.options.scope.join(" ");
                 console.log("Adding scope of " + scopeItems);
 
+                lock.withOptions {
+                    $0.oidcConformant = true
+                    $0.scope = scopeItems
+                }
+
                 //Pending update from telerik\progress
```

Comment:
@sitefinitysteve Request review here. I picked these from [Auth0 ios lock docs](https://auth0.com/docs/libraries/lock-ios/v2#oidc-conformant-mode). I'm not familiar with Swift to validate this.